engine/src/main/java/org/pentaho/di/www/ExecuteTransServlet.java

```diff
@@ -249,7 +249,7 @@ public void doGet( HttpServletRequest request, HttpServletResponse response ) th
 
     PrintWriter out = response.getWriter();
 
-    if ( repOption == null || transOption == null ) {
+    if ( transOption == null ) {
       response.setStatus( HttpServletResponse.SC_BAD_REQUEST );
       out.println( new WebResult( WebResult.STRING_ERROR, BaseMessages.getString(
               PKG, "ExecuteTransServlet.Error.MissingMandatoryParameter", repOption == null ? REP : TRANS ) ) );
```

Comment:
This result message needs to be updated too; if the repOption can be null, it shouldn't be reported as an error if it is.